zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -228,14 +253,29 @@
                 </group>
                 <names variable="editor translator" delimiter=", ">
                   <label form="verb-short" suffix=" "/>
-                  <name/>
+                  <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
                 </names>
               </if>
+              <else-if match="any" variable="translator">
+                <choose>
+                  <if match="none" variable="author">
+                    <names variable="editor">
+                      <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
+                      <label form="short" prefix=" (" suffix=")"/>
+                    </names>
+                  </if>
+                </choose>
+                <group delimiter=" ">
+                  <text variable="title" font-style="italic" suffix=","/>
+                  <text macro="translator"/>
+                  <text macro="collection-with-number"/>
+                </group>
+              </else-if>
```

Comment:
Don't debase yourself! Your updated style looks very good and I am only asking for some small nits here. I can change this issue here afterwards, if you prefer, but can you look over the terms and correct them?